### PR TITLE
test: Wait for dev server in tests

### PR DESCRIPTION
**Pull request by mshabarov**
Uses flow test utils to wait for the dev server to start.

### CODE REVIEW

**Comment by Artur-** [comment]
Sounds like TestBench is broken if this is needed?

File: pom.xml [hunk 1]
```diff
@@ -12,6 +12,7 @@
     <properties>
         <java.version>11</java.version>
         <vaadin.version>23.2-SNAPSHOT</vaadin.version>
+        <flow.version>2.7.15</flow.version>
```
Comment (Artur-) [inline pom.xml:15]: Flow version 2.7 for Vaadin 23.2 ?

File: pom.xml [hunk 2]
```diff
@@ -124,6 +125,12 @@
             <version>5.1.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>flow-test-util</artifactId>
```
Comment (Artur-) [inline pom.xml:130]: I don't think users of a starter will benefit from this dependency

File: pom.xml (hunk 1 above)
Comment (mshabarov) [reply pom.xml:15]: Ah, sorry, that was a rebase mistake. Actually the timing issue observed only with V14.8. Probably we miss some commit it TestBench

**Comment by mshabarov** [comment]
> Sounds like TestBench is broken if this is needed?

Agree. Better to check TestBench and find the problem there.

**Pull request closed without merging**